### ISSUE TITLE
printTypeDefinitions does not generate schema.gql when not passing an empty object

### ISSUE THREAD

**Issue by juan871003** [gatsbyjs/gatsby#26213]
## Summary

According to the documentation in [https://www.gatsbyjs.org/docs/actions/#printTypeDefinitions ]( https://www.gatsbyjs.org/docs/actions/#printTypeDefinitions) all parameters are optional.

I tried using `actions.printTypeDefinitions();` but it didn't generate the `schema.gql` file it should generate.
After some research I found out that I should pass an empty object like this: `actions.printTypeDefinitions({});`

I recommend stating this in the documentation, perhaps with an example:

```js
export const createSchemaCustomization = async ({ actions }) => {
    ...
    //  generates a file ./schema.gql
    actions.printTypeDefinitions({});
    ...
    actions.createTypes(...);
}
```

Some recommended topics to cover:
- https://www.gatsbyjs.org/docs/actions/#printTypeDefinitions 

### Motivation

Adding sample code makes documentation much clearer.

**Comment by meganesu** [comment]
Hi there @juan871003! Thanks for bringing this up. I agree that adding an example would help make these docs clearer.

I'm adding the Hacktoberfest label to this issue so that hopefully a contributor from our community can pick this up.

## Steps to resolve this issue

- [ ] Add an example to the docs for `printTypeDefinitions`, to show that the first object parameter is required (even though all the fields in the object are optional).

### Open a pull request

- [ ] Open a pull request with your work including the words "closes #26213" in the pull request description

**Comment by Kolkies** [comment]
Can I take this @meganesu?

**Comment by meganesu** [comment]
@Kolkies of course, go for it! Feel free to reach out if you have any questions.
